### PR TITLE
[ALLUXIO-1935] Use <> operator efficiently in BufferUtils.java

### PR DESCRIPTION
**Pull request by zhangsx123**
Use <> operator efficiently in BufferUtils.java.

### CODE REVIEW

**Comment by gpang** [comment]
Thanks for submitting a pull request to Alluxio!

We found your commits on GitHub aren't linked to a valid Github account yet (see your commits at https://github.com/Alluxio/alluxio/pull/3280/commits). 
For us to properly credit your contributions, could you please set up your Github account by following the Github instructions here (https://help.github.com/articles/setting-your-email-in-git/)?

Thank you again for your contribution to Alluxio!

**Comment by jsimsa** [comment]
Jenkins, ok to test

File: core/common/src/main/java/alluxio/util/io/BufferUtils.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/*
+﻿/*
```
Comment (jsimsa) [inline core/common/src/main/java/alluxio/util/io/BufferUtils.java]: undo this change

Comment (jsimsa) [inline core/common/src/main/java/alluxio/util/io/BufferUtils.java]: it is failing the build
